src/execution_tree/primitives/row_set.cpp

```diff
@@ -0,0 +1,360 @@
+            {
+                HPX_THROW_EXCEPTION(hpx::bad_parameter,
+                    "phylanx::execution_tree::primitives::"
+                    "row_set_operation::row_set_0d",
+                    generate_error_message(
+                        "use store operation for setting value to a variable",
+                        name_, codename_));
+            }
+
+            primitive_argument_type row_set1d(args_type&& args) const
+            {
+                auto row_start = args[1].scalar();
+                auto row_stop = args[2].scalar();
+                int step = args[3].scalar();
+                auto value_dimnum = args[4].num_dimensions();
+
+                if (step == 0)
+                {
+                    HPX_THROW_EXCEPTION(hpx::bad_parameter,
+                        "phylanx::execution_tree::primitives::"
+                        "row_set_operation::row_set1d",
+                        generate_error_message(
+                            "step can not be zero", name_, codename_));
+                }
+
+                if (value_dimnum == 2)
+                {
+                    HPX_THROW_EXCEPTION(hpx::bad_parameter,
+                        "phylanx::execution_tree::primitives::"
+                        "row_set_operation::row_set1d",
+                        generate_error_message(
+                            "can not store matrix in a vector", name_,
+                            codename_));
+                }
+
+                auto init_list = create_list_row_set(
+                    row_start, row_stop, step, args[0].size());
+
+                auto sv = blaze::elements(args[0].vector(), init_list);
+
+                if (value_dimnum == 0)
+                {
+                    blaze::DynamicVector<double> temp(
+                        sv.size(), args[4].scalar());
+                    sv = temp;
+                    return primitive_argument_type{ir::node_data<double>(0)};
+                }
+
+                auto temp = args[4].vector();
+
+                if (sv.size() != temp.size())
+                {
+                    HPX_THROW_EXCEPTION(hpx::bad_parameter,
+                        "phylanx::execution_tree::primitives::"
+                        "row_set_operation::row_set1d",
+                        generate_error_message("size mismatch, please check "
+                                               "your parameters or set vector",
+                            name_, codename_));
+                }
+
+                sv = temp;
+                return primitive_argument_type{ir::node_data<double>(0)};
+            }
+
+            primitive_argument_type row_set2d(args_type&& args) const
+            {
+                auto row_start = args[1].scalar();
+                auto row_stop = args[2].scalar();
+                auto step_row = args[3].scalar();
+                auto col_start = 0;
+                auto col_stop = args[0].dimensions()[1];
```

Comment:
`col_start` and `col_stop` will have different types, is that ok?

Reply:
Actually they were unused variables and I removed them from the code.